docs/markdown/Wrap-dependency-system-manual.md

```diff
@@ -284,6 +288,14 @@ With such wrap file, `find_program('myprog')` will automatically
 fallback to use the subproject, assuming it uses
 `meson.override_find_program('myprog')`.
 
+### CMake wraps
+
+Since the CMake module does not know the public name of the provided
+dependencies, a CMake `.wrap` file cannot use the `dependency_names = foo`
+syntax. Instead, the `dep_name = <target_name>_dep` syntax should be used, where
+`<target_name>` is the name of a CMake library with all non alphanum characters
+replaced by underscores `_`.
```

Comment:
It would be great to provide an example of this.

Reply:
done